src/quo2/components/navigation/page_nav.cljs

```diff
@@ -162,12 +163,13 @@
                        accessibility-label (assoc :accessibility-label accessibility-label
                                                   :accessible          true))
                      [button/button
-                      {:on-press       on-press
-                       :icon           (not label)
-                       :type           type
-                       :before         (when label icon)
-                       :size           32
-                       :override-theme icon-override-theme}
+                      {:on-press                  on-press
+                       :icon                      (not label)
+                       :type                      type
+                       :before                    (when label icon)
+                       :size                      32
+                       :override-theme            icon-override-theme
+                       :override-background-color icon-background-color}
```

Comment:
FYI: I used the exact same name `icon-background-color` that's used for the `left-section-view` component in this file. The prop was essentially missing.